pkg/webreg/webreg.go

```diff
@@ -706,6 +706,112 @@ func WebRegHandler(regAgent agents.RegistryAgent, confAgent agents.ConfigAgent)
 	}
 }
 
+// multiLineHighlightJS allows to append -$NUMBER to one of the line anchors
+// (e.G. `#line18-20`) and will then highlight all the lines.
+const multiLineHighlightJS = `<script>
+function getLinesToHighlight() {
+  let result = [];
+  url = window.location.href;
+  anchorSplitUrl = url.split("#");
```

Comment:
This creates global variables by accident, did you miss `let`?

Reply:
done